### PR TITLE
[14.0] edi: add disable flag in core mixin + exc type partner config

### PR DESCRIPTION
**Pull request by simahawk**
See atomic commits for details.

### CODE REVIEW

File: edi_oca/models/edi_exchange_consumer_mixin.py [hunk 1]
```diff
@@ -44,6 +44,12 @@ class EDIExchangeConsumerMixin(models.AbstractModel):
         default={},
     )
     edi_has_form_config = fields.Boolean(compute="_compute_edi_config")
+    # TODO: rename to `edi_disable_auto`
+    disable_edi_auto = fields.Boolean(
```
Comment (simahawk) [inline edi_oca/models/edi_exchange_consumer_mixin.py:48]: This is the old TODO I mentioned a long time ago. 
You can now reduce the override of this field only to the states.
CC @LoisRForgeFlow for edi_purchase and other modules.

Comment (simahawk) [reply edi_oca/models/edi_exchange_consumer_mixin.py:48]: @LoisRForgeFlow @etobella ping

Comment (LoisRForgeFlow) [reply edi_oca/models/edi_exchange_consumer_mixin.py:48]: Sorry @simahawk, were you expecting some action from our side? I thought it was more like a "FYI"

Comment (simahawk) [reply edi_oca/models/edi_exchange_consumer_mixin.py:48]: @LoisRForgeFlow yeah, just wanted an ack and maybe a review on the rest :wink:

**Comment by simahawk** [comment]
Tested on prod since a while, no objection here... let it go...

/ocabot merge minor

**Pull request merged**